### DISCUSSION TITLE
Any plans for integrating other providers?

### DISCUSSION THREAD

**Discussion by e3ndr** [Genuitec/Copilot4Eclipse-community#40] [Help] [answered]
In particular, I've been looking for an [ollama](ollama.com) eclipse extension (but an OpenAI integration would work just as well, since there are many projects which provide a OAI-compatible API using ollama for the inference) which would provide tab-completions. So far, I've found a project that has support for it, but it doesn't have tab completions and quite frankly has a clunky interface for interacting with the AI. Is this something that Genuitec would be interested in supporting or is it considered out-of-scope?

**Comment by wayneparrott** [comment, marked as answer]
@e3ndr We are open to new opportunities and have discussed supporting other LLMs in 2025. For the near term we plan to build out deeper integration of GitHub Copilot features into Eclipse. We have so many developer features we want to delivery asap.

Wayne
Copilot4Eclipse Team

**Comment by e3ndr** [comment]
Sounds awesome. I'll keep an eye out for it!

**Comment by reneleonhardt** [comment]
Any news on supporting local LLMs?
